index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <main id="main">
+      <h1 id="title">Dr. Norman Borlaug</h1>
+      <h3>The man who saved a billion lives</h3>
+      <div id="img-div">
```

Comment:
Please use semantic HTML tags like a section instead of a div, like <section id="img-div"> it makes clearly indicates the role of the content they contain